docs/src/guide/topology.md

```diff
@@ -164,22 +164,22 @@ Node pools enable you to create pools of machines within a cluster that have dif
 
 When using this module to create the node pools, the following is done:
 
-* a number of node pools are created. The number of node pools created is equal to the number of elements in the node_pools parameter e.g.
+* a number of node pools are created. The number of node pools created is equal to the number of elements in the worker_pools parameter e.g.
 
 ```
 worker_pools = {
-  np1 = { shape = "VM.Standard.E4.Flex", ocpus = 1, memory = 16, node_pool_size = 1, boot_volume_size = 150, autoscale = true, label = { app = "frontend", pool = "np1" } }
-  np2 = { shape = "VM.Standard.E4.Flex", ocpus = 1, memory = 16, node_pool_size = 1, boot_volume_size = 150, autoscale = true, label = { app = "frontend", pool = "np1" } }
+  np1 = { shape = "VM.Standard.E4.Flex", ocpus = 1, memory = 16, worker_pool_size = 1, boot_volume_size = 150, autoscale = true, label = { app = "frontend", pool = "np1" } }
```

Comment:
This can be `size` at the pool level, omitting the `worker_pool` prefix.

Reply:
Fixed